doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Beaglebone/index.md

```diff
@@ -5,4 +5,112 @@ permalink: /:path/
 sort_idx: 30
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The guide provides information on how to **cross-compile** C++ SDK for Beaglebone or Beaglebone Black.
+Alternatively, you can build the Kaa C++ endpoint SDK right on the Beaglebone board.
+Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/) for further details.
+
+## Install dependencies
+
+**The further instructions must be executed on the host machine.**
+
+**Verified against:**
+
+ - **Host OS:** Ubuntu 14.04 64-bit LTS
+ - **Target OS:** Ubuntu 16.04 32-bit LTS
+
+1. Install build prerequisites.
+
+        sudo apt-get install cmake build-essentials
+
+1. Download toolchain.
+
+        mkdir bb_root && cd bb_root
+        export BB_ROOT=$(pwd)
+        wget -c https://releases.linaro.org/components/toolchain/binaries/5.3-2016.02/arm-linux-gnueabihf/gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf.tar.xz
+        tar xf gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf.tar.xz
+        export CROSS=arm-linux-gnueabihf
+        export INSTALLDIR=$BB_ROOT/gcc-linaro-5.3-2016.02-x86_64_arm-linux-gnueabihf/libc/usr
```

Comment:
Would be better to rename `INSTALLDIR` to `SYSROOT`. (that's common name for such thing)